pkgs/tools/security/john/default.nix

```diff
@@ -8,11 +8,21 @@ stdenv.mkDerivation rec {
   pname = "john";
   version = "1.9.0-jumbo-1";
 
-  src = fetchurl {
-    url = "http://www.openwall.com/john/k/${pname}-${version}.tar.xz";
-    sha256 = "0fvz3v41hnaiv1ggpxanfykyfjq79cwp9qcqqn63vic357w27lgm";
+  src = fetchFromGitHub {
+    owner = "openwall";
+    repo = pname;
+    rev = "1.9.0-Jumbo-1";
```

Comment:
Should we rename the version string above?
The tag on GitHub is different from the release on the archive.

Reply:
Either way is fine, I think. What does the Jumbo-1 even mean anyway?

Reply:
*vernum-jumbo-n* means that this version is community enhanced (e.g. more ciphers can be used) version forked from *vernum* and the current development phase is *n*

Reply:
Ah like that. Yeah then it definitely sounds like it needs to stay